contrib/opencensus-ext-fastapi/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## Unreleased
+
+## 0.1.0
+Released 2022-05-04
```

Comment:
You can remove the `released` line. We will handle the release comments when we do release this.

Reply:
https://github.com/census-instrumentation/opencensus-python/pull/1124/commits/cbfb940c74768c3829788a6813ce38e24219eeaf